internal/app/machined/pkg/runtime/sequencer.go

```diff
@@ -91,6 +91,19 @@ func ParseSequence(s string) (seq Sequence, err error) {
 	return seq, nil
 }
 
+// ResetOptions are parameters to Reset sequence.
+type ResetOptions interface {
```

Comment:
fixing circular dependencies: abstract declares interface for the parameter to the reset sequence, v1alpha implements the interface